RetailCoder.VBE/UI/RubberduckUI.resx

```diff
@@ -2092,4 +2092,14 @@ Would you like to import them to Rubberduck?</value>
   <data name="CodeExplorer_AddModule" xml:space="preserve">
     <value>Add Module</value>
   </data>
+  <data name="CodeExplorer_EmptyViewMessage_Line1" xml:space="preserve">
+    <value>to parse and process</value>
+    <comment>Follows the text for "Refresh"</comment>
+  </data>
+  <data name="CodeExplorer_EmptyViewMessage_Line2" xml:space="preserve">
+    <value>the projects in the VBE.</value>
+  </data>
```

Comment:
Why does it have to be 2 resource strings? Wouldn't `\n` work?

Reply:
Didn't know that would work.  I'll change it.

Reply:
Actually, I think the text could be one a single line, and let the label wrap accordingly with the width of its parent element.

Reply:
Why use explicit formatting in the first place? Manually layouting in translations is generally speaking a hassle and usually not worth the effort

Reply:
The idea was to keep the text tight, and force a "wrap" where it makes grammatical sense in the sentence.  In other words, the brain naturally pauses before a conjunction ("the") and therefore it's easier to read when the second line starts there.

I thought about setting that textbox to a specific size, but then it would cause problems to do the same thing in another language.

A `\n` would certainly work.

Reply:
"where is makes grammatical sense" is introducing I18 concerns in the view itself, which isn't a good idea. Let the resource strings take care of that. Shift+Enter in the RESX editor should also work to make a multi-line resource string. Still, I'd let WPF itself deal with the wrapping if/when necessary, and leave the resource string on one line.